src/ui/Analytics/Analytics.ts

```diff
@@ -632,4 +634,15 @@ export class Analytics extends Component {
       return new Analytics(element, options, bindings).client;
     }
   }
+
+  private handleDoNotTrack() {
```

Comment:
Just wondering if we're going down the right path by supporting this.

According to MDN, the [doNotTrack property is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack), not recommended and might cease working at any time. It is also not supported by Safari.

Rather than embedding the functionality into the JSUI, could we ask for the client to wire it up themselves?

Reply:
Update:

Discussing with Olivier in slack, the idea of supporting this property is to show good faith to bots that decide what domains to block. By respecting `doNotTrack`, the hope is that bots would stop blocking Coveo's analytics endpoints.